canonical/canonical.com
# Create MongoDB pages

## Done

- Created pages for Mongodb
   - Refresh /data/mongodb [PR](https://github.com/canonical/canonical.com/pull/1348)
   - Created /data/mongodb/what-is-mongodb [PR](https://github.com/canonical/canonical.com/pull/1343)
   - Created /data/mongodb/support [PR](https://github.com/canonical/canonical.com/pull/1340)
   - Created /data/mongodb/managed [PR](https://github.com/canonical/canonical.com/pull/1352)
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- View all the pages to see if all the links are working correctly
- Demo
   - [/data/mongodb/](https://canonical-com-1355.demos.haus/data/mongodb/)
   - [/data/mongodb/what-is-mongodb](https://canonical-com-1355.demos.haus/data/mongodb/what-is-mongodb)
   - [/data/mongodb/support](https://canonical-com-1355.demos.haus/data/mongodb/support)
   - [/data/mongodb/managed](https://canonical-com-1355.demos.haus/data/mongodb/managed)
